OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -127,7 +127,7 @@ public WVec GetRepulsionForce(Actor other)
 
 		public Actor GetActorBelow()
 		{
-			if (self.CenterPosition.Z != 0)
+			if (self.World.Map.DistanceAboveTerrain(CenterPosition).Length != 0)
```

Comment:
Could you please add a comment here with your explanation about not using IsAtGroundLevel?